src/component/common/ConstraintAccordion/ConstraintAccordionEdit/StyledToggleButton/StyledToggleButton.tsx

```diff
@@ -0,0 +1,30 @@
+import { styled } from '@mui/system';
+import { IconButton } from '@mui/material';
+
+export const StyledToggleButtonOff = styled(IconButton)(({ theme }) => ({
+    width: 28,
+    minWidth: '28px!important',
+    minHeight: 40,
+    backgroundColor: 'white',
+    borderRadius: theme.shape.borderRadius,
+    padding: '0 1px 0',
+    marginRight: '1rem',
+    '&:hover': {
+        background: theme.palette.grey[300],
```

Comment:
Is there an abstraction we could use instead? Otherwise I'll have to fix that up in the dark mode PR.